client/src/inside/components/page/test/ehr-helper.spec.js

```diff
@@ -0,0 +1,227 @@
+import should from 'should'
+import EhrHelpV2, { LEAVE_PROMPT } from '../ehr-helper'
+import store from '../../../../store/index'
+import { getPageKeys } from '../../../../helpers/test/testHelper'
+
+// the following pageKey must contain one more table elements in its contents, 
+// so that it can be used in 
+const tablePageKey = getPageKeys()[3] 
+// the following pageKey must contain one more form elements in its contents, 
+// so that it can be used in 
+const formPageKey = getPageKeys()[0]
```

Comment:
"so that it can be used in"  ? what?  "in the following tests"?

Please explain the magic numbers 3 and 0